docs/background/gui-vs-api.md

```diff
@@ -0,0 +1,57 @@
+# Cleura Cloud Management Panel Vs. OpenStack API
+
+You can do many tasks in {{brand}} via the
+[{{gui}}](https://{{gui_domain}}) or via the OpenStack API, e.g., with 
+the help of the
+[OpenStack CLI](../../howto/getting-started/enable-openstack-cli) tool.
+For some tasks, though, you *will need* the OpenStack API.
+ 
+But when does it make sense to prefer a particular way of working?
+What are the reasons, if any, for choosing one method over the other?
+
+## The case for {{gui}}
+
+Ease of use is probably the main reason for choosing the {{gui}} over 
+the OpenStack API. Provided you have an [account in 
+{{brand}}](../../howto/getting-started/create-account), you can simply 
+log in and then follow step-by-step guides to create entities such as 
+[networks](../../howto/openstack/neutron/new-network), 
+[servers](../../howto/openstack/nova/new-server), or even 
+[Kubernetes clusters](../../howto/openstack/magnum/new-k8s-cluster). 
+You can just as easily perform administrative tasks like creating 
+[security 
+groups](../../howto/openstack/neutron/create-security-groups), setting 
+up [region-to-region VPN](../../howto/openstack/neutron/vpnaas) 
+connections, [deleting 
+networks](../../howto/openstack/neutron/delete-network), modifying 
+[billing data](../../howto/account-billing/change-billing-data), or 
+[managing invoices](../../howto/account-billing/manage-invoices).
+
+All in all, there are many instances when the {{gui}} is all you want 
+and, at the same time, is more than enough for what you want.
+
+## The case for OpenStack API
+
+There will be times when working from the terminal of your laptop is 
+preferable to using the {{gui}}. In cases like this, you will use the 
+OpenStack API via a tool like the OpenStack CLI. Although `openstack` 
+usually requires some reading before doing a specific task, it doesn't 
+take much time to get used to its logic for constructing commands to 
+achieve what you want. Plus, there is hardly a thing you can't do with 
+`openstack` or a CLI tool that talks to the OpenStack API. We should
+also point out that specific tasks can be performed _only_ via the
```

Comment:
```suggestion
`openstack` or a CLI tool that talks to the OpenStack API. 

We should
also point out that specific tasks can be performed _only_ via the
```
(I think we should break this paragraph into two.)

Reply:
Done!